### PR TITLE
Modernize Python 2 code to get ready for Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Make the minimal, safe changes required to convert the code to be syntax compatible with both Python 2 and Python 3. There would definitely be more work required to complete the port of the code to Python 3 but this is a minimal, safe first step.

    Run: futurize --stage1 -w **/*.py

See Stage 1: "safe" fixes http://python-future.org/automatic_conversion.html#stage-1-safe-fixes

$ pip install future
$ git checkout -b modernize-python2-code
$ futurize --stage1 -w **/*.py # done in zsh for ** globbing
$ git commit --all -m "Modernize Python 2 code to get ready for Python 3"
$ git push --set-upstream origin modernize-python2-code

### CODE REVIEW

File: deploy/ec2/ec2.py [hunk 1]
```diff
@@ -127,6 +127,7 @@
 These settings would produce a destination_format as the following:
 'webserver-ansible-blue-172.31.0.1'
 '''
+from __future__ import print_function
```
Comment (cutwater) [inline deploy/ec2/ec2.py]: Please move import below copyright comment.

File: galaxy/api/views.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```
Comment (cutwater) [inline galaxy/api/views.py]: ditto

File: galaxy/main/management/commands/refresh_role_counts.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```
Comment (cutwater) [inline galaxy/main/management/commands/refresh_role_counts.py]: ditto

Comment (cclauss) [reply galaxy/main/management/commands/refresh_role_counts.py]: Done.

File: galaxy/api/views.py (hunk 2 above)
Comment (cclauss) [reply galaxy/api/views.py]: Done.

**Pull request merged**